web/modules/custom/welcome/welcome.routing.yml

```diff
@@ -0,0 +1,7 @@
+welcome.welcomeCurrentUser:
+  path: '/welcome-user'
+  defaults: 
+    _controller: '\Drupal\welcome\Controller\WelcomeController::welcomeCurrentUser'
+    _title: 'This is the Welcome page'
+  requirements:
+    _permission: 'access content'
```

Comment:
Why use this when custom permission is created?

Reply:
I have fixed it.